R/acs_dm.R

```diff
@@ -156,10 +172,49 @@ dm.demand <- function(B17024,
   assertthat::assert_that(all(df$working_pov_rate <= 100, na.rm = TRUE))
   assertthat::assert_that(all(df$n_kids_lt5_working_under200_pct <= df$n_kids_lt6_working_under200_pct, na.rm = TRUE))
   assertthat::assert_that(all(df$n_kids_lt4_working_under200_pct <= df$n_kids_lt6_working_under200_pct, na.rm = TRUE))
-  
+
   return(df)
 }
 
+#' @title Add default attributes for ACS tables
+#' @inheritParams child_care_db
+acs_attr <- function(acs_year,
```

Comment:
I built out this function a little more. Ideally, I wanted to set up a similar function for each dataset that we were dealing with to manage the different parameter values.